api/src/main/java/ai/djl/util/Utils.java

```diff
@@ -105,17 +106,15 @@ public static void pad(StringBuilder sb, char c, int count) {
      * @param dir the directory to be removed
      */
     public static void deleteQuietly(Path dir) {
-        try {
-            Files.walk(dir)
-                    .sorted(Comparator.reverseOrder())
-                    .forEach(
-                            path -> {
-                                try {
-                                    Files.deleteIfExists(path);
-                                } catch (IOException ignore) {
-                                    // ignore
-                                }
-                            });
+        try (Stream<Path> stream = Files.walk(dir)) {
```

Comment:
This will cause dead lock, please change to the following:

```
        List<Path> list;
        try (Stream<Path> stream = Files.walk(dir)) {
            list = stream.sorted(Comparator.reverseOrder()).collect(Collectors.toList());
        } catch (IOException ignore) {
            // ignore
        }
        for (Path path : list) {
            try {
                Files.deleteIfExists(path);
            } catch (IOException ignore) {
                // ignore
            }
        }
```